main.tex

```diff
@@ -63,8 +82,21 @@
 
 \input{chapters/04_chapter_3.tex}
 
+\input{chapters/05_chapter_4.tex}
+
+\input{chapters/06_chapter_5.tex}
+
+\input{chapters/07_chapter_6.tex}
+
+\input{chapters/08_chapter_7.tex}
+
+\input{chapters/09_chapter_8.tex}
+
+\input{chapters/10_chapter_9.tex}
+
 \clearpage
 \bibliographystyle{unsrt}
-\bibliography{bibliography/english}
+%\addbibresource{bibliography/english}
+\bibliography{bibliography/ch9_fault_attacks}
```

Comment:
This overwrites the current bib. Please add your cites to the current file.

Reply:
Reverted